circadian/synthetic_data.py

```diff
@@ -0,0 +1,71 @@
+# AUTOGENERATED! DO NOT EDIT! File to edit: ../nbs/api/10_synthetic_data.ipynb.
+
+# %% auto 0
+__all__ = ['generate_activity_from_light']
+
+# %% ../nbs/api/10_synthetic_data.ipynb 4
+import matplotlib.pyplot as plt
+import numpy as np
+from .lights import LightSchedule
+
+# %% ../nbs/api/10_synthetic_data.ipynb 5
+def generate_activity_from_light(
+    time: np.ndarray, # time points.
+    light: LightSchedule, # light schedule.
+    mu_l: float=5.0, # mean of low activity levels.
+    mu_h: float=25.0, # mean of high activity levels.
+    mu_s: float=0.0, # mean of sleep activity levels.
+    sigma_l: float=7.5, # std of low activity levels.
+    sigma_h: float=30.0, # std of high activity levels.
+    sigma_s: float=2.0, # std of activity during sleep.
+    active_level: float=0.5, # person's activity level from 0 to 1.
```

Comment:
Maybe here's a good place to add in the docstring for this parameter that this is the probability of sampling from the high activity level vs sampling from the low activity level. Right?

Reply:
Good idea. I've added this info in a newer commit.

Reply:
Thanks!